fa_eval/README.md

```diff
@@ -1,22 +1,89 @@
-#### TimeFrame+Alignment evaluation code
+# TimeFrame+Alignment evaluation code
+## Theme: Forced Alignment 
+This is the alignment of the text to time positions for transcription/closed-captioning. 
+This tool processes human-annotated(`.tsv`) and machine-generated(`.mmif`) and outputs the evaluation results.
 
-This tool processes human-annotated(.srt) and machine-generated(.mmif) and outputs the evaluation results.
+To run the evaluation you will need matching prediction `.mmif` and gold `.tsv` files
 
-#### Parameters:
--m or --machine_dir: Directory containing machine annotated .mmif files.
+It is done by (tba).   
+It is evaluated by (tba). 
 
--o or --gold_dir: Directory containing human annotated .srt files.
+## Parameters
+-m or --machine_dir: Directory containing machine-annotated, model prediction `.mmif` files.  
+    * an existing set of predictions is [here](preds@gentle-forced-aligner-wrapper@aapb-collaboration-21-nongoldtext).  
+-g or --gold_dir: Directory containing human-annotated, golds `.tsv` files.   
+    * This is optional: not including the tag and its argument will default to using [`goldretriever.py`](goldretriever.py) to retrieve the golds 
+from the location specified in the `evaluate.py` script.  
+-r or --result_file: File path to store the evaluation results. Defaults to results.txt if not provided.    
+    * Suggested to always add this parameter to avoid overwriting.   
 
--r or --result_file: File path to store the evaluation results. Defaults to results.txt if not provided.
-
-#### Usage:
+## Environment
+It is recommended to set up a python environment for running the evaluation code:  
+Create a python env somewhere in your workspace:  
+```bash
+# in-that-workspace 
+python3 -m venv my_env_name
+source my_env_name/bin/activate
+# this will activate the python environment.
+# then run this to install the requirements for this evaluation. 
+pip3 install -r requirements.txt
+# now you may proceed to Usage.
+# After running the eval, deactivate your venv, run: `$ deactivate`. 
+```
 
-To run the evaluation code, run the following command:
+## Usage
+To run the evaluation code, run the following command while in the `fa_eval` directory:  
```

Comment:
I'd like to suggest, instead of this `Usage` section (which would be just duplicate to the CLI argument documentation), writing about the result output format as a `Output` section.